class.jetpack-gutenberg.php

```diff
@@ -27,6 +27,11 @@ function jetpack_register_block( $slug, $args = array() ) {
 		$slug = 'jetpack/' . $slug;
 	}
 
+	if ( isset( $args['version_requirements'] )
+			&& Jetpack_Gutenberg::is_block_version_gated( $args['version_requirements']['wp'], $args['version_requirements']['plugin'] ) ) {
+		return false;
+	}
+
```

Comment:
Should add an `unset( $args['version_requirements'] )` here, since we don't need to pass this to `register_block_type()`.